ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/AmountMapper.java

```diff
@@ -3,22 +3,27 @@
 import org.apache.commons.lang3.NotImplementedException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.commons.lang3.NotImplementedException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AUselessImportCheck)